app/db/interface.py

```diff
@@ -135,6 +166,37 @@ def _query_payments(self, payment_attributes):
         except exc.SQLAlchemyError:
             raise PaymentServiceQueryError('Could not retrieve payment items due to query error with given attributes')
 
+# UTILITY FUNCTIONS
+    def is_valid_put(self,old_data,new_data):
+        valid = False
```

Comment:
For now, is_valid_put takes 2 parameters - request data and user id. User id is needed to validate if there are any changes to user_id field and raises an error acc. Test cases for is_valid will be handled in another PR as mentioned above